### PR TITLE
Adapt webdriver.sh to Multiuser Che; add ability to set rerun attempts

### PR DESCRIPTION
**Pull request by dmytro-ndp**
### What does this PR do?
- moves `--multiuser` parameter from **selenium-test.sh** to **webdriver.sh**;
- fixed **webdriver.sh** script to compare selenium testing results with multiuser CI job;
- makes it clear the difference between 'CI job' from 'build' in terms of comparison of test results;
- adds an ability to set exact number of rerun of selenium tests.

### What issues does this PR fix or reference?
- #7511
- #7512

#### Docs PR
https://github.com/eclipse/che-docs/pull/323

### CODE REVIEW

File: selenium/che-selenium-core/bin/webdriver.sh [hunk 1]
```diff
@@ -50,21 +50,18 @@ initVariables() {
...
     [[ -z ${CUR_DIR+x} ]] && { CUR_DIR=$(cd "$(dirname "$0")"; pwd); }
 
     [[ -z ${API_SUFFIX+x} ]] && { API_SUFFIX="/api/"; }
-    [[ -z ${BASE_ACTUAL_RESULTS_URL+x} ]] && { BASE_ACTUAL_RESULTS_URL="https://ci.codenvycorp.com/view/qa/job/che-integration-tests-che6/"; }
 
     MODE="grid"
     GRID_OPTIONS="-Dgrid.mode=true"
-    RERUN=false
-
+    RERUN_ATTEMPTS=0
```
Comment (tolusha) [inline selenium/che-selenium-core/bin/webdriver.sh:64]: Why did you change the default value to `0` ?

File: selenium/che-selenium-core/bin/webdriver.sh [hunk 2]
```diff
@@ -412,13 +425,13 @@ HOW TO of usage:
...
         ${CALLER} <...> --failed-tests
-        ${CALLER} <...> --failed-tests --rerun
+        ${CALLER} <...> --failed-tests --rerun [ATTEMPTS]
 
     Debug selenium test:
         ${CALLER} -Mlocal --test=<TEST> --debug
 
     Analyse tests results:
-        ${CALLER} --compare-with-ci [CI job number]
+        ${CALLER} --compare-with-ci [BUILD NUMBER]
```
Comment (tolusha) [inline selenium/che-selenium-core/bin/webdriver.sh:434]: `BUILD NUMBER` collocation usually references to the product build number but `CI job number` indicates the number of the job on the CI.

File: selenium/che-selenium-core/bin/webdriver.sh [hunk 3]
```diff
@@ -428,7 +441,7 @@ printRunOptions() {
     echo "[TEST]"
     echo "[TEST] =========== RUN OPTIONS ==========================="
     echo "[TEST] Mode                : "${MODE}
-    echo "[TEST] Rerun failing tests : "${RERUN}
+    echo "[TEST] Rerun attempts      : "${RERUN_ATTEMPTS}
```
Comment (tolusha) [inline selenium/che-selenium-core/bin/webdriver.sh:444]: Does that mean all tests will be rerun ?

File: selenium/che-selenium-core/bin/webdriver.sh (hunk 1 above)
Comment (dmytro-ndp) [reply selenium/che-selenium-core/bin/webdriver.sh:64]: It means we DO NOT rerun tests if `--rerun` parameter hasn't been set.

File: selenium/che-selenium-core/bin/webdriver.sh (hunk 2 above)
Comment (dmytro-ndp) [reply selenium/che-selenium-core/bin/webdriver.sh:434]: You might be right. From this point, `[JOB BUILD]` sounds better, doesn't it?

File: selenium/che-selenium-core/bin/webdriver.sh (hunk 3 above)
Comment (dmytro-ndp) [reply selenium/che-selenium-core/bin/webdriver.sh:444]: Not, it doesn't. I have just realised that the code will be simpler if we replace `rerun = false` by identical by meaning `rerun attempts = 0`.

File: selenium/che-selenium-core/bin/webdriver.sh (hunk 1 above)
Comment (tolusha) [reply selenium/che-selenium-core/bin/webdriver.sh:64]: ok. So, let's automatically set `RERUN_ATTEMPTS` to `1` if `rerun` is set.
If a developer needs more attempts then he has to set it explicitly.

File: selenium/che-selenium-core/bin/webdriver.sh (hunk 2 above)
Comment (tolusha) [reply selenium/che-selenium-core/bin/webdriver.sh]: ok

Comment (tolusha) [reply selenium/che-selenium-core/bin/webdriver.sh:434]: Actually it is a job number.

File: selenium/che-selenium-core/bin/webdriver.sh (hunk 3 above)
Comment (tolusha) [reply selenium/che-selenium-core/bin/webdriver.sh:444]: ok

File: selenium/che-selenium-core/bin/webdriver.sh (hunk 2 above)
Comment (dmytro-ndp) [reply selenium/che-selenium-core/bin/webdriver.sh:434]: Job doesn't have a number. It has a name which is the name of the project to build https://wiki.jenkins.io/display/JENKINS/Building+a+software+project. And each build has unique number. 
For example, URL of page with test report https://ci.codenvycorp.com/view/qa/job/che-integration-tests-che6/100/testReport/ has:
- **che-integration-tests-che6**: name of project which job is aimed to build; 
- **100**: number of build of project. 

Therefore, we could name this number **100** as a `[PROJECT BUILD NUMBER]` and it can be confused with Che build namber, or as `[JOB BUILD NUMBER]` which is correct, but IMHO can be shorten to `[JOB BUILD]`.

Comment (tolusha) [reply selenium/che-selenium-core/bin/webdriver.sh:434]: ok, `BUILD_NUMBER` makes sense then.

File: selenium/che-selenium-core/bin/webdriver.sh (hunk 1 above)
Comment (dmytro-ndp) [reply selenium/che-selenium-core/bin/webdriver.sh:64]: Good advice, thank you!

**Review by tolusha — APPROVED** [review]
